clients/nexus-iq/src/main/kotlin/NexusIqService.kt

```diff
@@ -58,6 +58,7 @@ interface NexusIqService {
                     val requestBuilder = request.newBuilder()
                         .header("X-CSRF-TOKEN", token)
                         .header("Cookie", "CLM-CSRF-TOKEN=$token")
+                        .header("User-Agent", "ort-nexus-iq")
```

Comment:
As it's unrelated to adding OSS Index as an advisor, please split this out to a separate commit and explain in the commit message what this is good for.

Also, why does the user agent include the service name? I'd rather use a generic string like `.header("User-Agent", "$ORT_NAME/${Environment.ORT_VERSION}")` as we already do in [Downloader.kt](https://github.com/oss-review-toolkit/ort/blob/4be3ab1fbac441ae07fbc8f3f8399582975c0020/downloader/src/main/kotlin/Downloader.kt#L315). I might prepare a global variable for this.

Reply:
> I might prepare a global variable for this.

See https://github.com/oss-review-toolkit/ort/pull/3531.